src/Akka.Persistence.MongoDb/Query/QueryApi.cs

```diff
@@ -63,60 +63,63 @@ public EventAppended(string persistenceId)
         }
     }
 
-    /// <summary>
-    /// Subscribe the `sender` to current and new persistenceIds.
-    /// Used by query-side. The journal will send one <see cref="CurrentPersistenceIds"/> to the
-    /// subscriber followed by <see cref="PersistenceIdAdded"/> messages when new persistenceIds
-    /// are created.
-    /// </summary>
-    public sealed class SubscribeAllPersistenceIds : ISubscriptionCommand
+    [Serializable]
+    public sealed class SelectCurrentPersistenceIds : IJournalRequest
     {
-        /// <summary>
-        /// TBD
-        /// </summary>
-        public static readonly SubscribeAllPersistenceIds Instance = new SubscribeAllPersistenceIds();
-        private SubscribeAllPersistenceIds() { }
+        public IActorRef ReplyTo { get; }
+        public long Offset { get; }
+
+        public SelectCurrentPersistenceIds(long offset, IActorRef replyTo)
```

Comment:
Does this support an offset in Akka.Persistence.Sql @Arkatufus ?

Reply:
Yes, the long offset is supported